src/client/index.html

```diff
@@ -7,6 +7,15 @@
   <title><%= APP_TITLE %></title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+
+  <!-- TODO(girum): Remove Quickstart CSS Styles -->
+  <link href="https://www.gstatic.com/realtime/quickstart-styles.css"
```

Comment:
For the little text boxes that are in the GraphActivity. I'll remove this <link> tag soon.